files/docker/node/dockerfile_bin

```diff
@@ -38,15 +39,15 @@ RUN adduser --disabled-password --gecos '' guild \
     && adduser guild sudo \
     && mkdir -pv /home/guild/.local/ /home/guild/.scripts/ 
 
-
 RUN set -x && apt update \
     && apt-get update \
     && mkdir -p /root/.local/bin \
     && wget https://raw.githubusercontent.com/${G_ACCOUNT}/guild-operators/${GUILD_DEPLOY_BRANCH}/scripts/cnode-helper-scripts/guild-deploy.sh \
     && export SUDO='N' \
     && export UPDATE_CHECK='N' \
     && export SKIP_DBSYNC_DOWNLOAD='Y' \
-    && chmod +x ./guild-deploy.sh &&  ./guild-deploy.sh -b ${GUILD_DEPLOY_BRANCH} -s p \
+    && export G_ACCOUNT=${G_ACCOUNT} \
```

Comment:
What is the export of **G_ACCOUNT** intended to resolve? In `scripts/cnode-helper-scripts/guild-deploy.sh` **G_ACCOUNT** is changed to a hard coded **cardano-community**. Maybe this was missed in a cleanup and is no longer needed?

Reply:
hard coding 'cardano-community' causes decoupling of configuration files in fork dev env. please test cfg file changes in a fork to confirm.

Reply:
@Fuma419 I'd have cherry picked commit 852c1bf **_removing debug code_** for testing but it appears to be a mix of changes, including this export. 

After testing I do see your reasoning and that **G_ACCOUNT** was being reverted to `cardano-community` when `./guild-deploy.sh` gets executed. While not needed for upstream in, it is the sane choice to ensure the forked repository refers to itself properly when building the container. :+1: